tests/test_routes.py

```diff
@@ -0,0 +1,211 @@
+def test_get_all_planets_with_no_records(client):
+    # Act
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 200
+    assert response_body == []
+    
+def test_get_all_planets(client, two_planets):
+    response = client.get('/planets')
+
+    response_body = response.get_json()
+
+    assert response.status_code == 200
+    assert response_body == [{"id":1,
+        "name": "Earth",
+        "description": "home",
+        "moons": 1,
+        "life": True},
+        {"id":2,
+        "name": "Mars",
+        "description": "1st Colony",
+        "moons": 0,
+        "life": True}]
```

Comment:
We should be a little careful of assuming the order the planets come back in unless our route itself has an order by. Because we know that two planets were created in order by the fixture in a fresh database, it's mostly OK, but be aware that the database will really return the rows in whatever "internal" ordering happens to apply. We could use `in` checks for each planet record rather than assuming the ordering in a list to be a little more robust.